inductiva/tasks/task.py

```diff
@@ -225,6 +225,8 @@ def __str__(self):
             "\t" + line for line in data_metrics_table.splitlines())
 
         table_str = f"\nTask status: {self.status}"
+        if self.executer and self.executer.error_detail:
```

Comment:
I think this conditional statement should also check if task status is different from success.

Reply:
> I think this conditional statement should also check if task status is different from success.

I assume if we have error_details present they are relevant, even if the task was successful. I rather show it every time its present.